whats_left.sh

```diff
@@ -3,14 +3,10 @@ set -e
 
 ALL_SECTIONS=(methods modules)
 
-GREEN='[32m'
-BOLD='[1m'
-NC='(B[m'
-
 h() {
   # uppercase input
-  header_name=$(echo "$@" | tr "[:lower:]" "[:upper:]")
-  echo "$GREEN$BOLD===== $header_name =====$NC"
+  header_name=$(echo "$@")
```

Comment:
Maybe we could have an option for `whats_left.sh` that makes it output yaml, rather than doing it by default?

Reply:
sure, i will add an option for -o filename